testSpecs.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+SPECS_LOCATION=$(mktemp -d specs.XXXX)
+
+if test -z "$1"; then
+    SPECS_REF="master"
+else
+    SPECS_REF="$1"
+fi
+
+echo "travis_fold:start:generate_specs@$SPECS_REF"
+
+set -ev
+
+git clone --depth 1 --branch $SPECS_REF -- https://github.com/Azure/azure-rest-api-specs.git $SPECS_LOCATION
+
+for f in $(find $SPECS_LOCATION -iname README.md | grep -v node_modules); do 
+    autorest --go $f
```

Comment:
is autorest available as a global package ? it seems is getting pulled as a local dependency

Reply:
Good catch, fixed in my local.